protocol/message_persistence.go

```diff
@@ -68,7 +68,7 @@ func (db sqlitePersistence) tableUserMessagesAllFields() string {
 		sticker_hash,
 		image_payload,
 		image_type,
-		album_id,
+		image_album_id,
```

Comment:
This is a breaking change as well.
We need to make sure we have a migration in place that moves previous `album_id`s to `image_album_id`s

Reply:
@OmarBasem  if this is only a name change, I think we should just keep the old one in the database field, leave a comment if you wish, but migrations can be expensive and error prone, and still, we would have a dangling field since columns cannot be removed. Not worth the effort/risk in my opinion.

Reply:
@cammellos it is not just a name change. There is an albumId field on the Message struct and the ChatMessage struct (protobuf). Initially, both had the same name albumId, but then it was not working. I think they need to have different names, otherwise they would clash and the field would be nil (I think that's why it was not being delievered). It did not work until I changed the name on the Message struct to be ImageAlbumId.

Reply:
> @cammellos it is not just a name change. There is an albumId field on the Message struct and the ChatMessage struct (protobuf). Initially, both had the same name albumId, but then it was not working. I think they need to have different names, otherwise they would clash and the field would be nil (I think that's why it was not being delievered). It did not work until I changed the name on the Message struct to be ImageAlbumId.

That's a separate issue, we are only talking about the field in the database here, `image_album_id`, and that doesn't need to match the field in the structure.

In your case for example, is perfectly fine keeping `album_id` as a field in the database, but deserialize it into a field named `ImageAlbumID`

Reply:
Okay got it!